assembly/pom.xml

```diff
@@ -41,9 +41,6 @@
         <!-- Profile used to exclude console module on release process -->
         <profile>
             <id>console</id>
-            <activation>
```

Comment:
Because only `release:perform` takes in consideration additional profiles.
`release:prepare` uses the profiles given when you run the commands.